### PR TITLE
[EAGLE-4531]: 10.5.3 and avoid writing to dirs outside of CLI

### PR DESCRIPTION
**Pull request by zeiler**
Can't write to disk in imported functions as some file systems might be protected from writes. This is only needed for CLI usage so move to that part of the package.

### CODE REVIEW

File: clarifai/models/model_serving/cli/login.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import argparse
 
-from clarifai.utils.constants import CLARIFAI_PAT_PATH
+from clarifai.model_serving.constants import CLARIFAI_PAT_PATH
```
Comment (wemoveon2) [inline clarifai/models/model_serving/cli/login.py:3]: should this be `clarifai.models.model_serving.constants`?

File: clarifai/versions.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import os
 
-CLIENT_VERSION = "10.5.2"
+CLIENT_VERSION = "10.5.3"
```
Comment (wemoveon2) [inline clarifai/versions.py:3]: maybe this should read `VERSION` instead of hardcoding?

File: clarifai/models/model_serving/cli/login.py (hunk 1 above)
Comment (wemoveon2) [reply clarifai/models/model_serving/cli/login.py:3]: same with a few of the other imports

**Review by wemoveon2 — APPROVED** [review]
adjacent comments, can be ignored since it seems tests pass? or do we not have coverage?

File: clarifai/models/model_serving/cli/login.py (hunk 1 above)
Comment (zeiler) [reply clarifai/models/model_serving/cli/login.py]: good catch!

File: clarifai/versions.py (hunk 2 above)
Comment (zeiler) [reply clarifai/versions.py:3]: yeah I find this weird too. maybe @sanjaychelliah can take up

File: VERSION
[the remark this answers is not in the document]
Comment (zeiler) [reply VERSION:1]: yeah agreed, we do this in the grpc ones already.

File: clarifai/models/model_serving/cli/login.py (hunk 1 above)
Comment (wemoveon2) [reply clarifai/models/model_serving/cli/login.py]: oof... did we not have test coverage of this...???

**Pull request merged**